multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/services/ProcessLogsPersistenceService.java

```diff
@@ -78,6 +120,10 @@ public void persistLog(String space, String operationId, File localLog, String r
         }
     }
 
+    public void persistLog(OperationLogEntry operationLogEntry, DataSource dataSource) {
+        sqlOperationLogQueryProvider.saveLogInDatabase(operationLogEntry, dataSource);
+    }
```

Comment:
use the pattern like in other methods. They have rollback and close the transactions.